extras/data/machinery/generate/ATHENA.py

```diff
@@ -0,0 +1,49 @@
+from .utils import *
+import numpy as np
+
+
+def dump(src_path, target):
```

Comment:
Please merge the three `dump` (this one and the other two) with `dump_polarized` by adding argument and/or abstract workflow

Reply:
Hi Felix, could you clarify why? The reason that ATHENA, EIC and EIcC are not in dump_polarized is that their cases are completely custom, as the data is read from Yadism instead of commondata.

Reply:
mh? this has nothing to do with "data"* (in the NNPDF sense) - what you're supposed to do here is to prepare the yadism observable cards; for what you are using these is beyond `yadism` scope (i.e. if you use it just for the theory predictions or also for the pseudo data generatation). And just by looking at the code I can see that there is very strong overlap

PS: *fine calling this folder `data` is a bit misleading, maybe `real_life_experiments` would be better :upside_down_face:

Reply:
Okay let me clarify what I mean by data: the format of the projection datafiles inside Yadism is completely different from the other polarised sets (observe already the difference between .txt and .yaml). This is so, because all polarised (non-projective) data inside Yadism is directly computed from commondata and hence in similar format. If you really see the advantage of adding three distinct cases to the dump_polarized, I don't mind doing it, but I don't see the advantage myself to be honest.

Reply:
Look:
```
$ diff <(sed -n "68,104p;105q" utils.py) <(sed -n "6,40p;41q" ATHENA.py)
1,2c1,2
< def dump_polarized(src_path, target):
<     """Generate the input card for polarized measurements.
---
> def dump(src_path, target):
>     """Generate the input card for ATHENA measurements.
18,21c18,19
<     data = load(src_path, 0, ["x", "Q2"])
<     dict_kins = [
<         dict(x=d["x"]["mid"], y=d["y"]["mid"], Q2=d["Q2"]["mid"]) for d in data
<     ]
---
>     data = np.genfromtxt(src_path)
>     dict_kins = [dict(x=float(d[0]), y=float(d[2]), Q2=float(d[1])) for d in data]
```
the first two lines are completely irrelevant and for the second two I can see some trivial mapping. All the other 13 lines in that function are identical copies. Don't repeat yourself.